package.json

```
@@ -70,8 +89,11 @@
 		"prettier-plugin-packagejson": "2.5.3",
 		"prettier-plugin-sh": "0.14.0",
 		"release-it": "17.11.0",
+		"tsup": "^8.3.5",
+		"typescript": "^5.7.2",
```

I can't comment in `eslint.config.js`, but a good chunk of core ESLint rules are broken and/or redundant with TypeScript. This'll also need to add typescript-eslint there:

```suggestion
		"typescript": "^5.7.2",
		"typescript-eslint": "^8.19.0",
```

---

Do you have a preference with TS-ESLint config(s) I use?

---

`recommended` works for now. Eventually I'll (semi-automatically) migrate this to my template that applies `strictTypeChecked` and `stylisticTypeChecked`. 👍